lxd/containers.go

```diff
@@ -220,8 +217,11 @@ func containersShutdown(s *state.State) error {
 			wg.Wait()
 		}
 
+		// Record the current state
+		lastState := c.State()
+
 		// Stop the container
-		if c.IsRunning() {
+		if lastState != "BROKEN" && lastState != "STOPPED" {
```

Comment:
I don't know where the `BROKEN` state is coming from `liblxc` doesn't know this state iirc and `go-lxc` doesn't define it either. Did you mean `ABORTING` or `STOPPING`?

Reply:
It's a LXD state.

Reply:
That's what we return if we timeout getting a state from liblxc and likely what we'll be using when a cluster node is down.